core/src/wallet/tezos/TezosLikeAccount.cpp

```diff
@@ -143,7 +143,7 @@ namespace ledger {
                 if (OperationDatabaseHelper::putOperation(sql, operation)) {
                     emitNewOperationEvent(operation);
                 }
-                if (transaction.type == api::TezosOperationTag::OPERATION_TAG_ORIGINATION) {
+                if (transaction.type == api::TezosOperationTag::OPERATION_TAG_ORIGINATION && transaction.status == 1) {
```

Comment:
Do we have enum for this magic number?

Reply:
Yeah I raised that issue in some previous PRs, but we don’t currently. That’s on Tezos fault: they don’t provide semantics associated with the values if I recall correctly. :(

We could type it though.

Reply:
Same, but this would be impacting all the currencies if we want to do it properly, so I created a ticket for that : https://ledgerhq.atlassian.net/browse/LLC-488